content/events/2020-london/sponsor.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Sponsor
+---
+
+Kubernetes Community Days is a community-organized, not-for-profit conference series for practitioners.
+
+Sponsoring Kubernetes Community Days events gives your organization an opportunity to speak with practitioners, managers, and executives from companies of all sizes and industries. Whether your goals include recruiting new talent, opening new markets, or connecting with your existing customers, this event will be the place to have those conversations with experts and leaders in tech innovation.
+
+To enquire about sponsoring Kubernetes Community Days London 2020 get in touch via email at [sponsor@kuberentesdays.london](mailto:sponsor@kubernetesdays.london).
```

Comment:
I would encourage you to have the sponsor pricing listed here when you first publish.

Email has a typo.

Reply:
I'm still finalizing sponsorship, which might be another few weeks. I'd like to get the venue and date up sooner so we can start promoting the event sooner rather than later.